lib/structures/Guild.js

```diff
@@ -165,7 +165,7 @@ class Guild extends Base {
     }
 
     get iconURL() {
-        return this.icon ? `${CDN_URL}/icons/${this.id}/${this.icon}.${this.icon.startsWith("a_") ? "gif" : this.shard.client.options.defaultImageFormat}?size=${this.shard.client.options.defaultImageSize}` : null;
+        return this.icon ? `${CDN_URL}/icons/${this.id}/${this.icon}.${this.icon.startsWith('a_') ? 'gif':this.shard.client.options.defaultImageFormat}?size=${this.shard.client.options.defaultImageSize}` : null;
```

Comment:
Not sure why the cosmetic changes were made here, they go against the repo's eslint (double quotes are required, not sure about ternary spacing off the top of my head though)

Reply:
I honestly don't know, I'm used to using single quotes and never considered that. I'll change it to double quotes